dbt/project.py

```diff
@@ -95,10 +95,7 @@ def __init__(self, cfg, profiles, profiles_dir, profile_to_load=None,
             self.cfg['models']['vars'] = {}
 
         global_vars = dbt.utils.parse_cli_vars(getattr(args, 'vars', '{}'))
-        if 'vars' not in self.cfg['models']:
-            self.cfg['models']['vars'] = {}
-
-        self.cfg['models']['vars'].update(global_vars)
```

Comment:
why remove this and add global_vars below?

Reply:
Previously, this code updated the `cfg['models']['vars']` dict with vars defined on the CLI. These vars served as the "base" variables, and were overridden by other variables defined in the `models:` section of the `dbt_project.yml` file. My solution here was to store the cli vars separately, then merge them on top of any `dbt_project.yml`-based configs in the Var object. Can you think of a better way to do this?